### PR TITLE
chore: fix some SonarQube issues

### PR DESCRIPTION
**Pull request by Roming22**

### CODE REVIEW

File: pkg/engine/funcs_test.go [hunk 1]
```diff
@@ -6,17 +6,21 @@ import (
 	"testing"
 )
 
+var empty_input = "empty input"
+var valid_input = "valid input"
+var invalid_input = "invalid input"
```
Comment (otaviof) [inline pkg/engine/funcs_test.go]: Could you please move these variables into `TestToYAML` function and use camel-case? You may as well skip the word `var` by simply using `emptyInput := "empty input"` respectively.

**Review by otaviof — CHANGES_REQUESTED** [review]

File: pkg/engine/funcs_test.go (hunk 1 above)
Comment (Roming22) [reply pkg/engine/funcs_test.go]: As discussed, the vars need to be global. I did rename them to camel-case.

**Review by otaviof — APPROVED** [review]
/lgtm

**Review by otaviof — APPROVED** [review]
/lgtm

**Review by otaviof — APPROVED** [review]
/lgtm

**Pull request merged**